readme.md

```diff
@@ -26,7 +26,7 @@ githubUsername('sindresorhus@gmail.com', function (err, username) {
 
 ## API
 
-### githubUsername(email, [token], callback)
+### githubUsername(email, [token], [callback]): Promise<username>
```

Comment:
The correct notation would be `Promise<string>`, but I would just write this below instead:

> Returns a `Promise` for the username.